src/main/java/com/google/api/codegen/configgen/ProtoInterfaceTransformer.java

```diff
@@ -56,4 +58,9 @@
     }
     return nameMapBuilder.build();
   }
+
+  @Override
+  public void generateResourceNameGenerations(ConfigNode parentNode, ApiModel model) {
```

Comment:
I'm not sure yet what this does, but I'm a little wary of an interface method that requires a no-op implementation in one of its two major uses.

Reply:
We don't generate resource names from proto yet, so its implementation is a no-op. Filed #1783